recognition/include/pcl/recognition/linemod/line_rgbd.h

```diff
@@ -282,6 +282,10 @@ namespace pcl
       bool 
       readLTMHeader (int fd, pcl::io::TARHeader &header);
```

Comment:
You've removed the definition of this function

Reply:
`readLTMHeader` is only used in the old implementation of `loadTemplates`